crates/utils/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "vhost-device-utils"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+log = "0.4.19"
+thiserror = "1.0.40"
```

Comment:
drop (at least for now), probably it may make sense to report errors without panicking. But that should be a separate commit.